### PR TITLE
Updated the translation manual and fixed broken links

### PR DESCRIPTION
**Pull request by patjouk**
The guide was a bit outdated. There were also broken links. All fixed :smile_cat:

### CODE REVIEW

File: when_its_ready.md [hunk 1]
```diff
@@ -13,16 +13,15 @@ Here is an example PR for publishing a translation: https://github.com/DjangoGir
 Crowdin
 -------
 
-1. Go to the [Crowdin settings](https://crowdin.com/project/django-girls-tutorial/settings#translations) page.
-2. Click on the **Build project** button. If you don't have permissions to do that part, please email us at [hello@djangogirls.org](mailto:hello@djangogirls.org).
-3. After the build is done, click on the **Download** button next to the language you want to publish.
+1. You need one or two person to proofread and validate your translation. If no one on your team has proofreader status, please email us at [hello@djangogirls.org](mailto:hello@djangogirls.org).
```
Comment (phalt) [inline when_its_ready.md]: _one or two persons_ or _one or two people_ :)

**Comment by phalt** [comment]
:+1:

**Comment by patjouk** [comment]
I fix the typo and added one thing that was missing :tada:

**Comment by tricoder42** [comment]
That's perfect! Thank you for keeping this updated! :tada: :confetti_ball: :sparkles:

**Pull request merged**